dv/uvm/common_project_cfg.hjson

```diff
@@ -14,6 +14,10 @@
   scratch_path:       "{scratch_base_path}/{branch}"
   tool_srcs_dir:      "{scratch_path}/{tool}"
 
+  //Common DVSIM data structures
```

Comment:
Missing a space here

Reply:
Thanks for pointing that out, this should now be fixed.